docusaurus.config.js

```diff
@@ -1,106 +1,150 @@
-  url: 'https://docs.zeitgeist.pm',
-  baseUrl: '/',
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
-  favicon: 'img/favicon.ico',
-  organizationName: 'zeitgeistpm', 
-  projectName: 'documentation', // Usually your repo name.
-  themeConfig: {
-    navbar: {
-      title: 'Zeitgeist Documentation',
-      logo: {
-        src: 'img/Zeitgeist-trans.png',
+const config = {
+  title: "Zeitgeist Documentation",
+  tagline: "Learn how to integrate with the Zeitgeist network.",
+  url: "https://docs.zeitgeist.pm",
+  baseUrl: "/",
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "warn",
+  favicon: "img/favicon.ico",
+  organizationName: "zeitgeistpm",
+  projectName: "documentation", // Usually your repo name.
+
+  // support multi-languages
+  i18n: {
+    defaultLocale: "en",
+    locales: ["en", "zh-CN", "ru"],
+    localeConfigs: {
+      en: {
+        htmlLang: "en-GB",
       },
-      items: [
-        {
-          to: '/',  // getting-started',
-          label: 'Basics',
-          position: 'right',
-        },
-        {
-          to: 'prediction-markets',
-          label: 'Learn',
-          position: 'right',
-        },
-        {
-          to: 'guides',
-          label: 'Build',
-          position: 'right',
-        },
-      ],
-    },
-    footer: {
-      style: 'dark',
-      links: [
-        {
-          title: 'Community',
-          items: [
-            {
-              label: 'Website',
-              href: 'https://zeitgeist.pm',
-            },
-            {
-              label: 'Discord',
-              href: 'https://discord.com/invite/xv8HuA4s8v',
-            },
-            {
-              label: 'Twitter',
-              href: 'https://twitter.com/zeitgeistpm',
-            },
-          ],
-        },
-        {
-          title: 'More',
-          items: [
-            {
-              label: 'Blog',
-              to: 'https://blog.zeitgeist.pm',
-            },
-            {
-              label: 'GitHub',
-              href: 'https://github.com/zeitgeistpm',
-            },
-          ],
-        },
-      ],
-      copyright: `Copyright © ${new Date().getFullYear()} Zeitgeist PM`,
-    },
-    prism: {
-      additionalLanguages: ['bash', 'powershell'],
     },
   },
+
+  themeConfig:
+    /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
+    ({
+      navbar: {
+        hideOnScroll: true,
+        title: "Zeitgeist Documentation",
+        logo: {
+          src: "img/Zeitgeist-trans.png",
+        },
+        items: [
+          {
+            to: "/", // getting-started',
+            label: "Docs",
+            position: "left",
+          },
+          {
+            type: "docSidebar",
+            sidebarId: "sdk",
+            label: "SDK",
+            position: "left",
+          },
+          {
+            type: "docSidebar",
+            sidebarId: "guide",
+            label: "App Guide",
+            position: "left",
+          },
```

Comment:
I think we should delete these.

Generally, my plan for the wiki is to "steal" the style and structure of polkadot-wiki. This means that every chapter should be sorted into one of the "books", _General_, _Learn_, _Build_, _Maintain_. The SDK chapter and App chapters should not appear in the top bar, which must contain the books instead.

The polkadot-wiki has the books linked in the navbar. I don't think that's a good idea, since they are also available in the sidebar.